src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509Certificate.cs

```diff
@@ -460,6 +460,21 @@ public virtual byte[] GetSerialNumber()
             return serialNumber;
         }
 
+        /// <summary>
+        ///   Gets a value whose contents represent the big-endian representation of the
+        ///   certificate's serial number.
+        /// </summary>
+        /// <value>The big-endian representation of the certificate's serial number.</value>
+        public ReadOnlyMemory<byte> SerialNumberBytes
+        {
+            get
+            {
+                ThrowIfInvalid();
+
+                return GetRawSerialNumber();
```

Comment:
Do we care that the underlying byte[] can be extracted from the returned `ReadOnlyMemory<byte>` with MemoryMarshal.TryGetArray?  Or because we consider MemoryMarshal to be "unsafe", we're ok with needing that to access the internal state?

Reply:
I'm OK with it.  I can't think of any scenario other than malevolence where someone would be tempted to use TryGetArray with this value; and whoever would do that could do the same with private reflection.